src/components/options/comment/useDebugOptions.ts

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { faCopy } from "@fortawesome/free-regular-svg-icons";
+import { faBug } from "@fortawesome/free-solid-svg-icons";
+
+enum DebugOptions {
+  COPY_CONTENT_DATA = "COPY_CONTENT_DATA",
+  COPY_CHAIN_CONTENT_DATA = "COPY_CHAIN_CONTENT_DATA",
+}
+export const useDebugOptions = ({ comment: CommentType, commentChain }) => {
+  React.useMemo(() => {
+    return {
+      icon: faBug,
+      name: "Debug",
+      options: [
+        {
+          icon: faCopy,
+          name: "Copy content data",
+          link: {
+            onClick: () => {
+              copyText(comment.content);
+              toast.success("Data copied!");
+            },
+          },
+        },
+        {
+          icon: faCopy,
+          name: "Copy comment chain content data",
```

Comment:
Rather than having two separate options, I would check whether you've been passes a single comment or a chain, and then change the wording/click effect accordingly.